api/db/changelog/db.changelog-183-user-recent-resource-add-resource-type-identifier.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+    <changeSet author="nehasa" id="changelog-183-user-recent-resource-add-resource-type-identifier">
+        <addColumn tableName="user_recent_resource">
```

Comment:
Suggestion from @freemabd : Maybe use a new table with update script and use feature flag

Reply:
Closing this PR without merging, Will create a new PR with a new approach